packages/iTwinUI-react/src/core/Slider/Slider.tsx

```diff
@@ -24,9 +24,18 @@ export type TrackDisplayMode =
   | 'odd-segments'
   | 'even-segments';
 
-const getPercentageOfRectangle = (rect: DOMRect, pointer: number) => {
-  const position = getBoundedValue(pointer, rect.left, rect.right);
-  return (position - rect.left) / rect.width;
+const getPercentageOfRectangle = (
+  rect: DOMRect,
+  pointerX: number,
+  pointerY: number,
+  orientation: SliderProps['orientation'], // TODO: Confirm if this usage is correct
```

Comment:
Remove comment.

Reply:
Sorry, I added this comment/`TODO` to remind me to ask you if the parameter type I mentioned is correct. Can you please confirm if it is correct? Thanks

Reply:
Yup, it's ok.

Reply:
**[DONE]** Thanks